### ISSUE TITLE
[Rule Tuning] Endpoint rules using network data and event.type:connection

### ISSUE THREAD

**Issue by brokensound77** [elastic/detection-rules#419]
**tl;dr**: any rule using the index `logs-endpoint.events.*` (or `winlogbeat-*` and _can_ be compatible with endpoint) should not be using `event.type:connection` for network events.

## Description
There are many endpoint rules which are using `event.type:connection` for network data (`event.category:network`). The endpoint sensor does not populate `connection` to that field and so network rules would not be compatible across `winlogbeat-*` and `logs-endpoint.events.*` indexes.

Observed `network.protocol` correlations for 7.10 agents (per my limited testing dataset):
`event.type` | Endpoint | winlogbeat
-------------|----------|-----------
start | _missing_ | -, https, netbios-ns, llmnr, http, ssdp, domain, bootps, dhcpv6-server, ntp
end | _missing_ | _missing_
protocol | dns, http | -, https, netbios-ns, llmnr, http, ssdp, domain, bootps, dhcpv6-server, ntp, dns
info | dns | dns

`network.protocol` fields for `event.type:connection` in winlogbeat match up to `the protocol` fields, with both fields populated for backwards compatibility (it is a multi-value field)

Ideally both datasources will achieve parity in data and mappings, but until then, all impacted rules will need to have `connection` removed and have the `event.type` updated and logic adapted accordingly. We also should ensure ECS is being applied consistently/correctly across the datasources using these fields.
